olm/quickstarts/rhosak-openshift-quarkus-bind-quickstart.yaml

```diff
@@ -3,223 +3,230 @@ kind: ConsoleQuickStart
-  introduction: >-
-    ### This quick start shows you how to bind your Quarkus application to your Red Hat OpenShift Streams for Apache Kafka instance.
-
-    In this Quick Start, you'll bind a Quarkus application to your Kafka instance using Kubernetes Service Binding.
-    Service Binding allows you to communicate connection details and secrets to an application to allow it to bind to a service.
-    In this context, a service can be anything: a Kafka instance, a NoSQL database, etc.
-
-    By using Service Binding, we no longer need to configure connection details (host, port) authentication mechanisms (SASL, OAuth) and
-    credentials (username/password, client id/client secret) in an application. Instead, Service Binding injects these variables into your
-    application container (as files or environment variables), for your application to consume.
-
-    The Quarkus Kubernetes Service Binding extension enables Quarkus applications to automatically pickup these variables, injected as files, from the container's
-    filesystem, removing the need to specify any configuration settings in the application resources (e.g configuration files) themselves.
+    - You've completed the **Connecting OpenShift to Streams for Apache Kafka using the CLI** quick start.
+    - You have privileges to deploy applications in the OpenShift project that you connected your Kafka instance to.
+    - Your Kafka instance in [Streams for Apache Kafka](https://cloud.redhat.com/beta/application-services/streams/) is in the **Ready** state.
+
+  introduction: |-
+      ### This quick start shows how to use the Red Hat OpenShift Application Services (RHOAS) CLI to bind an example Quarkus application to a Kafka instance in Red Hat OpenShift Streams for Apache Kafka.
+
+      As a developer of applications and services, you can connect applications running on OpenShift to Kafka instances in Streams for Apache Kafka. Using an Operator to provide an application on OpenShift with the parameters required to connect to a Kafka instance is called *service binding*.
+
+      In this quick start, you'll use the RHOAS CLI and RHOAS Operator to bind an example Quarkus application to a Kafka instance. [Quarkus](https://quarkus.io/) is a Kubernetes-native Java framework that is optimized for serverless, cloud, and Kubernetes environments.
+
+      When you use the RHOAS CLI to bind a Kafka instance with an application, the RHOAS Operator automatically injects connection parameters as files into the Pod for the application. The example Quarkus application in this quick start uses the `quarkus-kubernetes-service-binding` [extension](https://quarkus.io/guides/deploying-to-kubernetes#service-binding), which means that the application automatically detects and uses the injected connection parameters. This eliminates the need for manual configuration of the application.
```

Comment:
The injection is not done by the RHOAS Operator. It's done by the Service Binding Operator.

Reply:
ACK, @DuncanDoyle.